content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -203,8 +271,16 @@ spec:
         clusterName: ${REMOTE_CLUSTER_NAME}
       network: ${REMOTE_CLUSTER_NETWORK}
 
-      # Replace ISTIOD_REMOTE_EP with the the value of ISTIOD_REMOTE_EP set earlier.
+      # Replaces ISTIOD_REMOTE_EP with the the value of ISTIOD_REMOTE_EP set earlier.
```

Comment:
I think this change is confusing. It seems to be saying that the following line of code replaces ISTIOD_REMOTE_EP, but I don't think that's what the code does. But I may be wrong or confused.

Reply:
```suggestion
      # Replace ISTIOD_REMOTE_EP with the the value of ISTIOD_REMOTE_EP set earlier.
```